bin/abbyy_catchup

```diff
@@ -26,13 +26,13 @@ successes, failures = results.partition(&:success?)
 # Manually process the successes and failures; continue on error
 successes.each do |result|
   watcher.process_success(result)
-rescue Dor::WorkflowException, Dor::Services::Client::Error => e
+rescue Dor::Services::Client::Error => e
```

Comment:
does anything replace `Dor::WorkflowException`, or is that just folded into Dor::Services::Client::Error in a post-dor-workflow-client world?